recipe/meta.yaml

```diff
@@ -16,6 +16,9 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
     - cython ==0.29.30
```

Comment:
The `cython` line is incorrect.  It should be listed in the `host` section (bare), and in the `build` section with the `# [build_platform != target_platform]` selector.

Reply:
doesn't look like the suggestion worked. I still get an import error. Now it occurs even on the linux build.